internal/datastore/spanner/migrations/zz_migration.0011_add_expiration_support.go

```diff
@@ -0,0 +1,35 @@
+package migrations
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+const (
+	addExpirationColumn = `
+		ALTER TABLE relation_tuple ADD COLUMN expires_at TIMESTAMP
+	`
+
+	addExpirationSupport = `
```

Comment:
please document in PR body that GC is hardcoded and cannot be changed, and open issue describing the gap